lib/options.go

```diff
@@ -321,7 +264,7 @@ type Options struct {
 	SummaryTimeUnit null.String `json:"summaryTimeUnit" envconfig:"summary_time_unit"`
 
 	// Which system tags to include with metrics ("method", "vu" etc.)
-	SystemTags TagSet `json:"systemTags" envconfig:"system_tags"`
+	SystemTags *stats.SystemTagSet `json:"systemTags" envconfig:"system_tags"`
```

Comment:
Given that this is just a simple integer now, I don't think we need a pointer to it

Reply:
@na–– you still need a pointer, to clarify whether user provide any tags or not.

Reply:
@na–– you still need a pointer, to clarify whether user provide any tags or not.

Reply:
Ah... yeah... :man_facepalming: OK, then it's probably better off as a pointer... I'm starting to intensely dislike dealing with these configuration issues in Go, Rust `enum`s are so much cleaner for something like this... :disappointed:

Reply:
@na-- done